inactivity.md

```diff
@@ -0,0 +1,36 @@
+---
+layout: default
+title: Maintainer and Repository Inactivity
+nav_order: 6
+---
+[//]: # (SPDX-License-Identifier: CC-BY-4.0)
+
+# Inactivity Policies
+
+As part of the normal lifecycle of a project codebases and maintainers come and go.  This document formalizes a Hyperledger wide polciy for archiving projects and moving maintainers to emereitus status when a proejct does not have an explicit policy of their own.
+
+## Maintainer Inactivity
+
+Hyperledger very much appreciates the contributions of all maintainers but removing write privledges is in the interest of an orderly and secure project.
+
+Maintainers who have not contributed to a Hyperledger project for (`exact timeframe to be decided`) 3 / 4 / 6 months will be removed from the relevant Github Organizations.
+
+Activity can be code contributions, code reviews, issue reporting, or any other such activity trackable by github attriubuted to a Hyperledger repository.
+
+Maintainers will notified of this via an "at" @ mention in a pull request moving their name from a projects active contributors to the emeritus contributors.
+
+Maintainers who express a desire to continue contributing will, at the maintainers request, be given a 3 month extension to contribute to the project.
+
+Maintainers who have been moved to emeritus status may be re-added by the projects when their contributions resume and the existing maintainers approve their reactivation.
+
+These policies apply to projects that do not have an expicity maintainer inactivity policy. Where a project has an established and functioning policy the projects policies shall prevail.
+
+## Repository Inactivity
+
+Any repository that has not had a release for (`exact timeframe to be decided`) 12 months or that has had no commits for 6 months may be archived by the TSC or Hyperledger staff.
```

Comment:
I think we need to reflect somehow that commits to be counted has to be of significant importance and not small updates and changes. Another point is that there are repos where maintainers extracted self contained functionality and there is no actually too much to update there besides probably some dependencies or release notes. IMO we also need to be careful with this here, I would consider per project health status rather than single repo.

Reply:
For a HLF wide policy we need a very objective policy, and judging the worth of a commit is about as subjective as it gets.  Individual projects can make their own policy if they want to judge the significance of a contribution.

Regarding the static "framework" repositories, the clause about requests not to close has no limit (unlike the contributor policy) so a true framework can be kept unarchived indefinitely, until the security notifications start piling up.

One reason to focus this per-repository instead of per-project is that some project repos have been forgotten by their projects when maintainers leave. If the repo is valuable the team can request it not be archived.